File: 0485-max-consecutive-ones/0485-max-consecutive-ones.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int b=0,c=0,max=0;
    while(b<numsSize){
        if(nums[b]==1){
            c++;
        }
        else if(c>max){
            max=c;
            c=0;
        }
        else if(nums[b]==0)
            c=0;
        b++;
    }
    if(c>max)
        max=c;
    return max;
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int a=0,b=0,c=0,max=0;
        while(b<nums.length){
            if(nums[b]==1){
                b++;
                c++;
            }
            else{
                max=Math.max(max,c);
                c=0;
                a=b+1;
                b=a;
            }
        }
        max=Math.max(max,c);
        return max;
    }
}